src/main.qml

```diff
@@ -51,10 +51,33 @@ Application {
         icon: "ios-sync"
         visible: !btStatus.connected
     }
+
     Item {
         anchors.fill: parent
         visible: btStatus.connected
 
+        Image {
+            id: coverArt
+            anchors.fill: parent
+            fillMode: Image.PreserveAspectCrop
+            opacity: 0.7
+            source: if (mprisManager.currentService) {
+                var artTag = Mpris.metadataToString(Mpris.ArtUrl)
+                return (artTag in mprisManager.metadata) ? mprisManager.metadata[artTag] : ""
+            }
+        }
+
+        Rectangle {
+            visible: coverArt.status == Image.Ready
```

Comment:
Worth giving it a name to remember what it is here for. Something like:

`id: textBackgroundDarkener`